### PR TITLE
Adding N45GH, A57061 as Walmart aircraft that was seen

### PR DESCRIPTION
**Pull request by challgren**
## Describe your changes

PLANE INFO: Walmart N45GH A57061

## Checklist before requesting a review

- [X] I have performed a self-review of my code.
- [ ] While waiting for someone to review your request, please help review [another open pull request](https://github.com/sdr-enthusiasts/plane-alert-db/pulls) to support the maintainers.

### CODE REVIEW

**Review by Phaeton — APPROVED** [review]

**Comment by dziban303** [comment]
There are way more than 12849 planes in the db now

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ This project consists of lists of 'interesting' aircraft, formatted as CSV files
 
 ## Current Content
 
-There currently are about **12848** unique aircraft in **49** categories found in this repository. This [Dashboard](https://datastudio.google.com/reporting/eb19ab53-b622-4946-b34a-9667232c136d/page/4taCC) contains details of the main list and the most recent additions.
+There currently are about **12849** unique aircraft in **49** categories found in this repository. This [Dashboard](https://datastudio.google.com/reporting/eb19ab53-b622-4946-b34a-9667232c136d/page/4taCC) contains details of the main list and the most recent additions.
```
Comment (Phaeton) [inline README.md:19]:
```suggestion
There currently are about **13243** unique aircraft in **49** categories found in this repository. This [Dashboard](https://datastudio.google.com/reporting/eb19ab53-b622-4946-b34a-9667232c136d/page/4taCC) contains details of the main list and the most recent additions.
```

**Review by Phaeton — CHANGES_REQUESTED** [review]

**Review by Phaeton — APPROVED** [review]

**Comment by Phaeton** [comment]
> There are way more than 12849 planes in the db now. We've not been updating that figure in the readme. Not much point in incrementing it by one, is there?
> 
> Looks like there's 13242 lines in the main db.

I updated it to 13243, current db +1. Need to look at an action that autoupdates the README.

**Comment by challgren** [comment]
> > There are way more than 12849 planes in the db now. We've not been updating that figure in the readme. Not much point in incrementing it by one, is there?
> > Looks like there's 13242 lines in the main db.
> 
> I updated it to 13243, current db +1. Need to look at an action that autoupdates the README.

Yeah I figured it happened automatically but wasn't 100 sure. Hopefully I'll contribute more.

**Comment by Phaeton** [comment]
Thanks for the contribution!

**Pull request merged**